vdev/src/commands/compile_vrl_wasm.rs

```diff
@@ -0,0 +1,25 @@
+use std::path::Path;
+use anyhow::Result;
+use clap::Args;
+
+use crate::app;
+
+/// Compiles VRL crates to wasm32-unknown-unknown
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let crates = vec!["compiler", "core", "diagnostic", "parser"];
+        let vrl_path = Path::new(app::path()).join("lib").join("vrl");
+        let args = vec!["build", "--release", "--target", "wasm32-unknown-unknown"];
+
+        for crate_name in crates {
```

Comment:
I don't think you really need vecs for either of these:
```suggestion
        let vrl_path = Path::new(app::path()).join("lib").join("vrl");
        let args = &["build", "--release", "--target", "wasm32-unknown-unknown"];

        for crate_name in  ["compiler", "core", "diagnostic", "parser"] {
```
(and no `clone` on the args below)